Janrain/JRConnectionManager/JRConnectionManager.m

```diff
@@ -41,7 +41,9 @@
 @implementation NSString (JRString_UrlEscaping)
 - (NSString *)stringByAddingUrlPercentEscapes
 {
-    NSString *encodedString = [self stringByAddingPercentEncodingWithAllowedCharacters:[NSCharacterSet URLQueryAllowedCharacterSet]];
+    NSMutableCharacterSet *allowed = [[NSCharacterSet URLQueryAllowedCharacterSet] mutableCopy];
```

Comment:
I considered making a static `NSCharacterSet *` and using `dispatch_once` wrapped in a nil check to assign it, but wasn't sure if that would be overkill.